cmake/lzhl.cmake

```diff
@@ -0,0 +1,13 @@
+set(LZHL_DIR ${CMAKE_SOURCE_DIR}/Dependencies/.lzhl)
```

Comment:
You'll need to add an entry for the .gitignore to ignore this folder or you'll have people committing this in PRs and such accidentally. Same for zlib.

Reply:
I blanket ignored all things starting with dot already, so this is ignored.